### PR TITLE
Prepare for tenancy - add tenants table and tenant_id to all current tables

### PR DESCRIPTION
**Pull request by gtanzillo**

### CODE REVIEW

**Comment by agrare** [comment]
Should we create at least a simple tenant model also?  Maybe something we can lookup from a userid string and then set the tenant_id in refresh.

**Comment by gtanzillo** [comment]
> Should we create at least a simple tenant model also? Maybe something we can lookup from a userid string and then set the tenant_id in refresh.

Makes sense, I'll add that

File: db/migrate/20181005182645_add_tenant_and_tenant_id_to_all_tables.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class AddTenantAndTenantIdToAllTables < ActiveRecord::Migration[5.0]
+  create_table "tenants", id: :bigserial, force: :cascade do |t|
```
Comment (agrare) [inline db/migrate/20181005182645_add_tenant_and_tenant_id_to_all_tables.rb]: Shouldn't this be in the `change` method?

Comment (gtanzillo) [reply db/migrate/20181005182645_add_tenant_and_tenant_id_to_all_tables.rb]: Doh, you're right. I threw it in last minute. I'll take care of that and also add the model.

**Comment by gtanzillo** [comment]
@agrare This on should be ready to go. Drew and I worked out the schema for the `tenants` table.

**Pull request merged**